packages/server-wallet/e2e-test/pong/controller.ts

```diff
@@ -49,18 +49,11 @@ export default class PongController {
       channelId: calculateChannelId({...convertedSignedStates[0]}),
     });
 
-    if (channelResult.turnNum < 4) {
-      // TODO: Join Channel
-      return {
-        recipient: from,
-        sender: this.myParticipantID,
-        data: {signedStates: [], objectives: []},
-      };
-    }
-
     const {
       outbox: [messageToSendToPing],
-    } = await this.wallet.updateChannel(channelResult);
+    } = await (channelResult.turnNum < 4 ? this.wallet.joinChannel : this.wallet.updateChannel)(
```

Comment:
Does `joinChannel` actually get called in `e2e-test.ts` as a result of this change?

Reply:
Yeah, but the `turnNum` does not increase

Reply:
You should still be able to assert that the channel now has `{supported: {turnNum: 0}}`, instead of `{supported: undefined}`

Reply:
Good point, I'll add this.